src/test/acceptance/features/navigation.feature

```diff
@@ -36,7 +36,7 @@ Feature: Application process navigation is controlled
     Then I am shown the <application page> page
     Examples:
       | application page    | navigation page |
-      | enter name          | card address    |
+      | enter name          | address    |
```

Comment:
Minor, can you align the formatting of the final pipe please?

Reply:
Updated formatting.